src/main/update/extract.ts

```diff
@@ -0,0 +1,57 @@
+import extractZip from "extract-zip";
```

Comment:
Maybe we should consider some generic library for this: https://www.npmjs.com/package/decompress. I guess we can discuss this in another isssue.

Reply:
In the first step, I did try to use the existing scripts
As you said, we should need to create a separate issue